### PR TITLE
Ol Onal

### PR DESCRIPTION
**Pull request by eggrobin**
[[173-C26](https://www.unicode.org/cgi-bin/GetL2Ref.pl?173-C26)] Consensus: Accept 44 Ol Onal characters at U+1E5D0..U+1E5FF, with names, properties, and glyphs as given in document [L2/22-151R](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/22-151R), in a new Ol Onal block that extends from U+1E5D0..U+1E5FF for a future version of the Unicode Standard. (Ref. section 5 of [L2/22-248](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/22-248)).

Checked consistency with Ken’s draft UnicodeData (d8) and LineBreak (d3).

### CODE REVIEW

**Comment by markusicu** [comment]
Hi @josh-hadley in your absence we were handing out script encoding pull requests, and this one looks “not too difficult” so we are giving it to you to review :-)

**Comment by eggrobin** [comment]
Made the nasalization mark a diacritic, and made the vowel length mark a diacritic extender.

File: unicodetools/data/ucd/dev/ScriptExtensions.txt [hunk 1]
```diff
@@ -618,17 +618,17 @@ A830..A832    ; Deva Dogr Gujr Guru Khoj Knda Kthi Mahj Mlym Modi Nand Shrd Sind
 
 # ================================================
 
-# Script_Extensions=Beng Deva Dogr Gong Gonm Gran Gujr Guru Knda Mahj Mlym Nand Orya Sind Sinh Sylo Takr Taml Telu Tirh
+# Script_Extensions=Beng Deva Dogr Gong Gonm Gran Gujr Guru Knda Mahj Mlym Nand Onao Orya Sind Sinh Sylo Takr Taml Telu Tirh
```
Comment (josh-hadley) [inline unicodetools/data/ucd/dev/ScriptExtensions.txt:621]: I am probably missing something, but L2/22-151R has it as `Olon`, not `Onao` — why is it different?

Comment (eggrobin) [reply unicodetools/data/ucd/dev/ScriptExtensions.txt:621]: @markusicu works in mysterious ways :-)

Comment (markusicu) [reply unicodetools/data/ucd/dev/ScriptExtensions.txt:621]: I don't always follow the suggestions for script codes, just like we don't always follow suggested property values.
In this case, the approved code fits established patterns for existing codes.

Comment (markusicu) [reply unicodetools/data/ucd/dev/ScriptExtensions.txt:621]: Another answer is that the authoritative list of script codes is at https://www.unicode.org/iso15924/iso15924-codes.html and the recent changes are easy to see at https://www.unicode.org/iso15924/codechanges.html

Comment (josh-hadley) [reply unicodetools/data/ucd/dev/ScriptExtensions.txt:621]: OK. The actual code used doesn't matter to me, I just wanted to connect the dots on the difference. I'm satisfied with these explanations 😸

**Review by josh-hadley — DISMISSED** [review]

**Comment by markusicu** [comment]
FYI @josh-hadley remember that Robin had to merge main changes in here which dismissed your approval, so please rubber-stamp...

**Review by josh-hadley — APPROVED** [review]

**Pull request merged**